OpenOversight/app/main/views.py

```diff
@@ -675,7 +675,6 @@ def redirect_add_department():
 @admin_required
 def add_department():
     form = DepartmentForm()
-    form.created_by = current_user.get_id()
```

Comment:
This was in `add_department`, but it certainly doesn't belong there as the field no longer exists.

<img width="995" alt="Screenshot 2023-08-30 at 1 31 33 PM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/c4da97b3-f716-4d06-843d-59cf7caf976d">